Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "sparse_vec"
+description = "Compress vectors using row displacement"
+version = "0.1.0"
+authors = ["Lukas Diekmann <lukas.diekmann@gmail.com>"]
+repository = "https://github.com/softdevteam/sparsevec/"
+readme = "README.md"
+# This should be "Apache-2.0 OR MIT OR UPL-1.0" but crates.io doesn't know about
+# UPL-1.0.
+license = "Apache-2.0 OR MIT"
+
+[dependencies]
+vob = "1.3.2"
```

Comment:
In general you only want to specify `x.y` since the `.z` (in this case ".2") should be compatible. But since there's a newer Vob, I'd suggest putting `2.0` here.